cmd/estuary-shuttle/main.go

```diff
@@ -75,6 +76,72 @@ func init() {
 	}
 }
 
+func overrideSetOptions(flags []cli.Flag, cctx *cli.Context, cfg *config.Shuttle) error {
```

Comment:
I know this method is temp, but I think you should move it under the shuttle config (`cfg.overrideWithDefaults(flags []cli.Flag, cctx *cli.Context)`), to reduce noise on `main.go` and to allow for easy testing.

Reply:
I do see your concerns -- my view is more that command line processing is a primary function of main, and actually it's almost everything else that should be moved out. On first reading of the code, my instinct was that app.Action is way to long, and there should be another file containing a simple runEstuary(config.Estuary) that does the heavy lifting.  From a testing perspective I think this is better as it lets test code stand up an estuary instance without getting involved in any command line issues. Let's bounce it around Slack for consensus? I'll do it your way if that's what everyone else thinks.